.github/workflows/unittest.yml

```diff
@@ -23,14 +23,17 @@ jobs:
     - name: Install dependencies
       run: |
         wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
-        bash miniconda.sh -b -p $HOME/miniconda
+        bash miniconda.sh -b -p $HOME/miniconda || true
```

Comment:
This will always force this line to be true, when testing locally, there is an issue where the miniconda will already exist and will fail on subsequent test runs.